x-pack/plugins/security_solution/server/endpoint/endpoint_app_context_services.ts

```diff
@@ -196,7 +196,12 @@ export class EndpointAppContextService {
   public async getEndpointAuthz(request: KibanaRequest): Promise<EndpointAuthz> {
     const fleetAuthz = await this.getFleetAuthzService().fromRequest(request);
     const userRoles = this.security?.authc.getCurrentUser(request)?.roles ?? [];
-    return calculateEndpointAuthz(this.getLicenseService(), fleetAuthz, userRoles);
+    return calculateEndpointAuthz(
+      this.getLicenseService(),
+      fleetAuthz,
+      userRoles,
+      this.startDependencies?.appFeaturesService
```

Comment:
Wondering if we should first check for `this.startDependencies?.appFeaturesService` and if `undefined`, then `throw`?  This would ensure that we must use `appFeatureService` on the server side when checking `authz`. _(unless `appFeaturesService` is not available in no-serverless deployments (I don't remember) - then just disregard my comment)_